### PR TITLE
Ptv 1766 fix interleaved check

### PR DESCRIPTION
**Pull request by jkbaumohl**
I added a check to the FASTQ import to make sure the line count was a multiple of 8 (since 4 lines per read) for interleaved files.

### CODE REVIEW

**Review by MrCreosote — DISMISSED** [review]
LGTM. This must be a pretty rare error since this code is 6+ years old...

File: lib/ReadsUtils/ReadsUtilsImpl.py [hunk 1]
```diff
@@ -875,7 +875,11 @@ def validateFASTQ(self, ctx, params):
                 t.flush()
                 shutil.copy2(t.name, file_path)
             validated = 1
-            if c % 4 != 0:
+            if p.get('interleaved') and c % 8 != 0:
```
Comment (MrCreosote) [inline lib/ReadsUtils/ReadsUtilsImpl.py:878]: This needs tests

**Review by MrCreosote — CHANGES_REQUESTED** [review]

**Comment by MrCreosote** [comment]
Also tests are failing

File: lib/ReadsUtils/ReadsUtilsImpl.py (hunk 1 above)
Comment (jkbaumohl) [reply lib/ReadsUtils/ReadsUtilsImpl.py:878]: I did test all permutations on CI. I will see if I can figure out how to add a test in for it.  We also did not have a test for the %4 before, from what I can tell.

Comment (MrCreosote) [reply lib/ReadsUtils/ReadsUtilsImpl.py:878]: This might be it: https://github.com/kbaseapps/ReadsUtils/blob/master/test/ReadsUtils_server_test.py#L638

Comment (MrCreosote) [reply lib/ReadsUtils/ReadsUtilsImpl.py:878]: That being said, this whole module needs a rewrite. It was the first SDK module ever AFAIR and my code quality was not high.

Comment (jkbaumohl) [reply lib/ReadsUtils/ReadsUtilsImpl.py:878]: Ok I added a test.

**Comment by MrCreosote** [comment]
Code and the new test looks good, although the crumminess of the API makes writing a good test difficult. However there's still the issue that the tests are failing

**Review by jkbaumohl — COMMENTED** [review]
The google drive ticket that is likely causing these tests to fail https://kbase-jira.atlassian.net/browse/PUBLIC-1962

**Comment by MrCreosote** [comment]
Can you add a TODO to the commented out tests with a link to the ticket so future people know why the tests were commented out? Other than that looks good.

**Comment by MrCreosote** [comment]
Hum, the tests are failing again.

**Comment by jkbaumohl** [comment]
So it appears something intermittent with blobstore (says Shock but that is old).  First time I ran the tests again on CI I got the same issue, and it bailed out fairly quickly.  I ran it again and everything passed.  ----------------------------------------------------------------------
Ran 111 tests in 1280.450s

OK

**Comment by jkbaumohl** [comment]
Saw it failed again. There is something temperamental going on here, not sure what. I don't think the issues lie within readutils.

**Comment by MrCreosote** [comment]
It's consistently failed all 5 times within 4 minutes

**Review by MrCreosote — APPROVED** [review]
I ran the tests locally and they passed first time. Now I think of it, I remember other people having problems with GHA tests that contact outside servers

**Pull request merged**